code/ulab.h

```diff
@@ -357,6 +357,10 @@
 #define ULAB_LINALG_HAS_INV             (1)
 #endif
 
+#ifndef ULAB_LINALG_HAS_SOLVE_TRIANGULAR
```

Comment:
Could you, please, move it a bit lower, so that the functions are in alphabetical order (at least at the module level)?